core/src/main/scala/org/apache/spark/status/api/v1/ApplicationListResource.scala

```diff
@@ -69,6 +69,8 @@ private[spark] object ApplicationsListResource {
       attempts = app.attempts.map { internalAttemptInfo =>
         new ApplicationAttemptInfo(
           attemptId = internalAttemptInfo.attemptId,
+          startTimeEpoch = internalAttemptInfo.startTime,
```

Comment:
Yeah, you would certainly have to expose getters for the new values. You just add them to the class declaration. This should work?

```
class ApplicationAttemptInfo private[spark](
  ...) {
  ...
  def getStartTimeEpoch: Long = startTime.getTime
  ...
}
```